### DISCUSSION TITLE
OpenAI Model GPT-4 support

### DISCUSSION THREAD

**Discussion by WilliamLin** [FlowiseAI/Flowise#753] [Q&A]
Hello, I'm interested in utilizing gpt-4 with the OpenAI LLM, but I'm unable to find the gpt-4 LLM model in the dropdown menu. I also noticed that gpt-3.5-turbo isn't listed either
![OpenAI Model Support](https://github.com/FlowiseAI/Flowise/assets/1085796/de16925b-6e32-4138-b080-c5db311608e3)
. Is there a way for me to use gpt-4 with FlowisesAI?

Appreciate your help.

**Comment by chungyau97** [comment]
Hi @WilliamLin,

Correct me if I'm wrong, 
[GPT 4](https://platform.openai.com/docs/models/gpt-4) is using Chat Completions endpoint.
[Completions](https://platform.openai.com/docs/api-reference/completions/object) endpoint such as LLM `OpenAI` node will become legacy.
Try using `ChatOpenAI` node under Chat Models.

**Comment by Farscaper** [comment]
It shows up for me as mentioned in the Chat OpenAI node. Use that instead.
https://share.getcloudapp.com/lluXKngY

**Comment by WilliamLin** [comment]
Thanks for the feedback. I am able to get GPT 4.0 working now.
